services/payments/tests/unit/conftest.py

```diff
@@ -150,3 +161,55 @@ def _cancel_200(request: httpx.Request):
         ).mock(side_effect=_cancel_200)
 
     return _mock
+
+
+def pytest_addoption(parser: pytest.Parser):
+    group = parser.getgroup(
+        "external_environment",
+        description="Replaces mocked services with real ones by passing actual environs and connecting directly to external services",
+    )
+    group.addoption(
+        "--external-envfile",
+        action="store",
+        type=Path,
+        default=None,
+        help="Path to an env file. Consider passing a link to repo configs, i.e. `ln -s /path/to/osparc-ops-config/repo.config`",
```

Comment:
nice!

Reply:
the CLI was your idea ;-) . I gave you credit for it :-)